pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -249,6 +249,22 @@
         "hideFromDocs": true
       }
     },
+    {
+      "metadata": {
+        "name": "alertingDeletePermanently",
+        "resourceVersion": "1742988550168",
+        "creationTimestamp": "2025-03-26T11:29:10Z"
+      },
+      "spec": {
+        "description": "Enables the UI functionality to delete permanently alert rules",
+        "stage": "GA",
+        "codeowner": "@grafana/alerting-squad",
+        "frontend": true,
+        "hideFromAdminPage": true,
+        "hideFromDocs": true,
+        "expression": "true"
+      }
+    },
```

Comment:
nit: What about naming it `alertingRulePermanentlyDelete`? And for the description:
`Enables UI functionality to permanently delete alert rules`